Rarime/Code/Modules/ScanPassport/ViewModels/PassportViewModel.swift

```diff
@@ -173,67 +200,14 @@ class PassportViewModel: ObservableObject {
                 return proof
             }
             
-            var isUserRevoking = passportInfo.activeIdentity != Ethereum.ZERO_BYTES32
 #if DEVELOPMENT
             if DebugManager.shared.shouldForceRegistration {
                 isUserRevoking = false
             }
 #endif
+            let isUserRevoking = false
```

Comment:
add a comment that revokation is disabled and keep the old code line as a comment